core/src/main/java/com/scalar/db/transaction/consensuscommit/CrudHandler.java

```diff
@@ -114,6 +120,12 @@ public void delete(Delete delete) {
 
   private Scanner getFromStorage(Scan scan) throws CrudException {
     try {
+      // only get after image columns
```

Comment:
```suggestion
      // get only after image columns
```
Just for consistency 🙇

Reply:
Ah sorry. I missed it. Thank you!